### DISCUSSION TITLE
Keep popover open while opening a modal

### DISCUSSION THREAD

**Discussion by Newbie012** [chakra-ui/chakra-ui#7865] [Q&A]
### Description

I've encountered an issue where a popover contains a form with a file input. When I click on the file input, a modal with a drag-and-drop area opens. However, opening the modal results in the active popover closing.

As a temporary workaround, I tried using closeOnBlur={false}, but it leads to a less intuitive user experience. With this setting, clicking outside both the popover and the modal doesn't close the popover, which isn't ideal. I would expect the popover to close in this scenario.

### Problem Statement/Justification

Written above

### Proposed Solution or API

I'm not sure whether the popover or the modal should be responsible for handling this case.

Assuming that the popover will be responsible for not being closed, I would go with this approach:

```
<Popover closeOnFocusPortal={false} ... />
```

### Alternatives

As written above, I could use `closeOnBlur={false}` but it's not ideal.

### Additional Information

I think the issue is while using `<Portal />` in general (which is being used when creating a new `<Modal />`).

**Comment by lenngri** [comment]
I have the same issue. I would be nice to be able to combine Popovers and Modals as described above!

**Comment by shoeferlin** [comment]
+1, same issue

**Comment by TrystanHumann** [comment]
Has this been addressed?
